### DISCUSSION TITLE
Any further update or release on this?

### DISCUSSION THREAD

**Discussion by RickVincent** [jheddings/notional#387] [General]
Hi Jason,

Just curious, **any further update or release** on this great API project?

Thanks. /RickVencent

**Comment by jheddings** [comment]
Thanks for reaching out!  I've been working on an update to Pydantic v2 in my limited spare time.  It's a pretty big lift, so I have not been adding to the existing library.

Is there a specific feature or capability you'd like to see?

**Comment by RickVincent** [comment]
Happy to see your plan on upgrading to Pydantic v2 or even v3. My apps based on this is broken now.
And, would be great if can support page lock/unlock feature like: https://github.com/jamalex/notion-py#example-lockunlock-a-page.
